letter.md

```diff
@@ -1,25 +1,21 @@
-The Comprehensive R Archive Network (CRAN), is unusual among user-contributed software package infrastructure due to the high standards to which contributions are held, and the continuous checking system that validates packages against reverse dependencies and operating systems. 
+The Comprehensive R Archive Network (CRAN) is exceptional infrastructure. Thanks to high standards for contributions and a thorough continuous checking system, users can trust that any package from CRAN is likely to work, be compatible with other CRAN packages, and leave the local R package library in a consistent state. These guarantees are fundamental to the success and popularity of R, and they would not be possible without the tireless effort and diligence of the CRAN team.
```

Comment:
I like this opening.

'tireless effort' doesn't sit well with me. There's evidence to suggest some of them are getting tired and corners are getting cut.

Maybe just "...without the diligence of the CRAN team."

Reply:
Implemented the change.